controllers/mustgather/validation_test.go

```
@@ -716,3 +719,304 @@ func Test_validateSFTPWithRetry(t *testing.T) {
+		}
+		if !strings.HasPrefix(receivedAuth, "Basic ") {
+			t.Errorf("expected Basic auth, got %q", receivedAuth)
+		}
+	})
+
+	t.Run("proxy auth with special characters uses raw credentials", func(t *testing.T) {
+		ln, err := net.Listen("tcp", "127.0.0.1:0")
+		if err != nil {
+			t.Fatalf("failed to start listener: %v", err)
+		}
+		defer ln.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `ln.Close` is not checked

(errcheck)